components/ScheduleTable.tsx

```diff
@@ -69,22 +69,24 @@ export default function ScheduleTable({
             <Tr key={event.title}>
               <Td>
                 <Flex flexDirection="column">
-                  <Text isTruncated>
+                  <Text isTruncated align="center">
                     {event.start.toLocaleTimeString([], {
                       hour: "2-digit",
                       minute: "2-digit",
                     })}
                   </Text>
                   {mobile && <Text align="center">-</Text>}
-                  <Text isTruncated>
+                  <Text isTruncated align="center">
                     {event.end.toLocaleTimeString([], {
                       hour: "2-digit",
                       minute: "2-digit",
                     })}
                   </Text>
                 </Flex>
               </Td>
-              <Td>{event.title}</Td>
+              <Td>
+                <Text align="center">{event.title}</Text>
```

Comment:
Nitpick: I think the event title should be left aligned to make it easier to read (being lined up on left makes it faster to read from one line to another since the starting position is always in the same place)

Reply:
HAHA I originally only had it centered for the smallest breakpoint and left aligned for the rest, but I wanted to see if you'd say something if I centered them all LOL